recipes/hkl/yum_requirements.txt

```diff
@@ -0,0 +1,5 @@
+# gtk-doc provides "gtkdocize" needed during the building process
+gtk-doc
```

Comment:
This seems like the sort of thing that should be a conda-forge package and listed in the build or host requirements

Reply:
Can you please point which package would provide that? I didn't find any.

Reply:
It seems like it is just called gtk-doc: https://www.gtk.org/gtk-doc/download.php

Reply:
I saw that. Is there already a corresponding conda package for that? With this PR I didn't intend to build that package, but I can look into it.

Reply:
There doesn't seem to be a package for it yet: https://anaconda.org/search?q=gtk-doc

If you want to skip this step of the build, you can get away with not having this dep :wink:

Reply:
The corresponding patch and the flags to `configure` exclude building docs for now.